### PR TITLE
Adding waffling60 Rev D ISO Hotswap

### PR DESCRIPTION
**Pull request by 4pplet**
## Description

Adding the first PCB with the new revision of waffling60 Rev D. Project moves from ATmega32u2 to APM and STM32F072 due to the issue with stock/availability and age of the 32u2. Project will be split into Hotswap ISO, Hotswap ANSI and Solder. ANSI and Solder code will be added when prototyped and tested.

Thanks! <3

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/4pplet/waffling60/rev_d_iso/config.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#pragma once
```
Comment (drashna) [inline keyboards/4pplet/waffling60/rev_d_iso/config.h:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/4pplet/waffling60/rev_d_iso/rev_d_iso.c [hunk 2]
```diff
@@ -0,0 +1 @@
+#include "rev_d_iso.h"
```
Comment (drashna) [inline keyboards/4pplet/waffling60/rev_d_iso/rev_d_iso.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/4pplet/waffling60/rev_d_iso/rev_d_iso.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#pragma once
```
Comment (drashna) [inline keyboards/4pplet/waffling60/rev_d_iso/rev_d_iso.h:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by 4pplet** [comment]
Thanks! License information added @drashna

**Review by drashna — COMMENTED** [review]
Since there appears to be only a single layout defined, could you change `LAYOUT_all` to just `LAYOUT`?

**Review by fauxpark — APPROVED** [review]

File: keyboards/4pplet/waffling60/info.json [hunk 4]
```diff
@@ -1,80 +0,0 @@
-{
-    "manufacturer": "4pplet",
-    "url": "https://github.com/4pplet/waffling60",
-    "maintainer": "4pplet",
-    "usb": {
-        "vid": "0x4444"
```
Comment (drashna) [inline keyboards/4pplet/waffling60/info.json:6]: Looks like moving/removing this breaks the previous revisions.

Comment (4pplet) [reply keyboards/4pplet/waffling60/info.json:6]: My mistake! Should be fixed now ^^

**Review by drashna — APPROVED** [review]

**Comment by fauxpark** [comment]
@4pplet Configurator keymaps also need updating.
image

**Comment by 4pplet** [comment]
Thanks, I'll look into this! :)

**Pull request merged**